frontend/src/CreateBook.tsx

```diff
@@ -48,11 +49,11 @@ export default function CreateBook(props: ModalProps) {
     }
 
     return (
-        <Modal
-            isOpen={props.modalIsOpen}
-            onRequestClose={props.closeModal}
-            contentLabel="Example Modal"
-            ariaHideApp={false}
+        <Modal className="modal"
```

Comment:
This removes the styles that the React modal already brings with it. I found them useful for a start.
Consider removing the class and setting custom styles with the class:
.ReactModal__Content--after-open